documentation/dsl.md

```diff
@@ -1055,83 +1118,57 @@ PARTITION(Addresses, by=state)
 PARTITION(People, name="ppl")
 ```
 
-**Bad Example #4**: Count how many packages were ordered in each year. Invalid because `YEAR(order_date)` is not allowed ot be used as a partition term (it must be placed in a CALC so it is accessible as a named reference).
+**Bad Example #4**: Count how many packages were ordered in each year. Invalid because `YEAR(order_date)` is not allowed ot be used as a partition term (it must be placed in a `CALCULATE` so it is accessible as a named reference).
```

Comment:
```suggestion
**Bad Example #4**: Count how many packages were ordered in each year. Invalid because `YEAR(order_date)` is not allowed to be used as a partition term (it must be placed in a `CALCULATE` so it is accessible as a named reference).
```